cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1484,6 +1487,9 @@ def _add_compute_resource_launch_template(
                 instance_market_options=instance_market_options,
                 instance_initiated_shutdown_behavior="terminate",
                 capacity_reservation_specification=capacity_reservation_specification,
+                metadata_options=ec2.CfnLaunchTemplate.MetadataOptionsProperty(http_tokens="required")
```

Comment:
Unit tests should cover the setting of this metadata.

Reply:
At this moment afaik we do these tests by asserting on the produced Cfn we generate with CDK (see `test_required_imds_v2`)